pcsx2-qt/QtHost.h

```diff
@@ -77,6 +78,9 @@ class EmuThread : public QThread
 	void stopBackgroundControllerPollTimer();
 	void updatePerformanceMetrics(bool force);
 
+	/// Filters mouse wheel events from unfocused combo boxes, spin boxes and sliders.
+	MouseWheelFilter MouseWheelFilterObj;
```

Comment:
Like I said above, just create the object in the dialog and parent it (that'll take care of destroying it when the parent is destroyed).

No need to keep a reference around, because it's never accessed later, and it's definitely not related to the EmuThread.

Reply:
Alright.

Reply:
The idea was `g_emu_thread` was always alive, so I latched an instance of the filter onto it.

Reply:
and also the filter gets installed into the widget, not the other way around, so destroying the widgets like QComboBoxes...etc causes the filter to get "uninstalled" since it gets destroyed along with the widget.

Reply:
Other than memory leak concerns, is it still preferable creating filter objects in the parents of the widgets?

> and it's definitely not related to the EmuThread.

Reply:
As far as I'm aware, child widgets should be destroyed in reverse order to what they're added in, so it shouldn't be an issue.

> parents of the widgets?

I was suggesting in the dialog, not the widgets.

Reply:
yes I was referring to the dialog by "parent".